packages/common/src/api/README.md

```diff
@@ -0,0 +1,150 @@
+# audius-query
+
```

Comment:
Maybe add a quick high level overview describing audius-query and it's benefits

Reply:
Might also be good to include a quick description of how the entities relate to the cache and get normalized

Reply:
Great ideas. Thanks!